diff --git a/pl_mpeg.h b/pl_mpeg.h
index 4e69dfca5d..a270bf8c91 100755
--- a/pl_mpeg.h
+++ b/pl_mpeg.h
@@ -4133,12 +4133,12 @@ void plm_audio_decode_frame(plm_audio_t *self) {
 							? self->samples.left
 							: self->samples.right;
 						for (int j = 0; j < 32; j++) {
-							out_channel[out_pos + j] = self->U[j] / 2147418112.0f;
+							out_channel[out_pos + j] = self->U[j] / 1090519040.0f;
 						}
 					#else
 						for (int j = 0; j < 32; j++) {
 							self->samples.interleaved[((out_pos + j) << 1) + ch] = 
-								self->U[j] / 2147418112.0f;
+								self->U[j] / 1090519040.0f;
 						}
 					#endif
 				} // End of synthesis channel loop
