diff --git a/pl_mpeg.h b/pl_mpeg.h
index 4e69dfca5d..71b7edda92 100755
--- a/pl_mpeg.h
+++ b/pl_mpeg.h
@@ -1386,12 +1386,13 @@ int plm_buffer_no_start_code(plm_buffer_t *self);
 int16_t plm_buffer_read_vlc(plm_buffer_t *self, const plm_vlc_t *table);
 uint16_t plm_buffer_read_vlc_uint(plm_buffer_t *self, const plm_vlc_uint_t *table);
 
+
 plm_buffer_t *plm_buffer_create_with_filename(const char *filename) {
-	FILE *fh = fopen(filename, "rb");
-	if (!fh) {
+	// FILE *fh = fopen(filename, "rb");
+	// if (!fh) {
 		return NULL;
-	}
-	return plm_buffer_create_with_file(fh, TRUE);
+	// }
+	// return plm_buffer_create_with_file(fh, TRUE);
 }
 
 plm_buffer_t *plm_buffer_create_with_file(FILE *fh, int close_when_done) {
